java/src/com/ibm/streamsx/topology/internal/context/remote/ZippedToolkitRemoteContext.java

```diff
@@ -180,33 +196,68 @@ private static Path pack(final File appTkRoot, JsonObject submission, String tkN
                 }
             }
         }
-        
+
+        if (deploy.has(ContextProperties._SPLMM_OPTIONS)) {
+            List<String> splmmOptions;
+            JsonElement opts = deploy.get(ContextProperties._SPLMM_OPTIONS);
+            if (opts.isJsonArray()) {
+                splmmOptions = new ArrayList<>();
+                for (JsonElement e : opts.getAsJsonArray()) {
+                    splmmOptions.add(e.getAsString());
+                }
+            } else {
+                splmmOptions = Collections.singletonList(opts.getAsString());
+            }
+            
+            if (!splmmOptions.isEmpty()) {
+                try (PrintWriter splmmOptsW = new PrintWriter(splmmOptsTmp.toFile(), "UTF-8")) {
+                    for (String splmmOpt : splmmOptions) {
+                        splmmOptsW.print(splmmOpt);
+                        splmmOptsW.print(" ");
+                    }
+                }
+            }
+
+            if (splmm_dir[0] != "") {
```

Comment:
This needs to be `!= null` right?

Reply:
Oh, I keep forgetting to run `ant`, sorry, will fix